projects/core/src/i18n/translation-chunk.service.ts

```diff
@@ -35,7 +35,7 @@ export class TranslationChunkService {
     return chunk;
   }
 
-  private warnDuplicates(items: { [key: string]: string[] }): void {
+  protected warnDuplicates(items: { [key: string]: string[] }): void {
```

Comment:
I am not sure if this should be protected as it does not say in the documentation nothing about that for services.

Note: I did this as well, and asked Wei for her opinion and said to keep it private and conservative since we can always change it later on.

Reply:
ok If Wei said I do